client/app/create-recipes/create-recipes-controller.js

```diff
@@ -0,0 +1,34 @@
+angular.module('wtf.create-recipes', [])
+  .controller('CreateRecipesController', function($scope, $window, Auth, Recipe) {
+
+    $scope.recipe = {};
+
+    $scope.$watch(Auth.isAuth, function(authed) {
+      if (authed) {
+        $location.path('/create-recipes');
+      } else {
+        $location.path('/');
+      }
+    }, true);
+
+    $scope.root = 
+
+    $scope.submit = function(ingredient) {
+      Recipe.createRecipe(recipe).then(function(data, err) {
+        if (err) console.log(err);
+        $scope.getData();
+        $scope.recipe = {};
+      });
+    };
+
+    $scope.getData = function() {
```

Comment:
Since this is the create view, I don't think that the $scope.getData method will be needed.